uiowa/uiowa
# Parallel deployment updates processing

Resolves #7800 

# How to test
- To avoid running updates against all sites locally, make sure you have a `multisites` entry set up in `local.blt.yml`.
- Test first without parallelization:
```
ddev blt artifact:update:drupal:all-sites
```
- Set up parallelization:
```
ddev ssh
```
- Inside DDEV:
```
sudo apt install parallel && exit
```
Note: This will be removed with a ddev update/restart.
- Test with parallelization:
```
ddev blt artifact:update:drupal:all-sites
```
- Mangle some config: make an edit to any config file in the default config set so that the YAML is invalid. This will cause site deployment updates to fail.
```
ddev blt artifact:update:drupal:all-sites
```
- After all output from parallel deployments, you should see `[error]  Error deploying updates. Check the log output for more information. `
